python/cudaq/kernel/ast_bridge.py

```diff
@@ -1365,6 +1365,22 @@ def bodyBuilder(iterVal):
                 opCtor([], [], [], [qubitA, qubitB])
                 return
 
+            if node.func.id == 'u3':
+                # Single target, three parameters `u3(θ,φ,λ)`
```

Comment:
Aside: at some point we should decide as a group how to write these angles. Code has gone from greek letters, to a mix of greek letters and English spellings of greek letters, to fully spelling them out in English (e.g. "angle_one"). I don't have a preference, but I don't think we need churn for the sake of churn.

Reply:
Tried to address in [d013595](https://github.com/NVIDIA/cuda-quantum/pull/1504/commits/d013595eacb390d5e6eb6b8044a1c7a1c325d7c3)
Did not change the comments within the code.